tests/integration_test/core_http_system_test.c

```diff
@@ -0,0 +1,629 @@
+/*
+ * FreeRTOS V202011.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file core_http_system_test.c
+ * @brief Integration tests for the coreHTTP library, for the FreeRTOS platform
+ * over the Secure Sockets transport layer.
+ */
+
+/* Standard header includes. */
+#include <string.h>
+#include <stdbool.h>
+#include <assert.h>
+#include <stdlib.h>
+#include <time.h>
+
+/* Unity Test framework include. */
+#include "unity_fixture.h"
+
+/* Kernel includes. */
+#include "FreeRTOS.h"
+#include "task.h"
+
+/* Include header for root CA certificates. */
+#include "iot_default_root_certificates.h"
+
+/* Include paths for public enums, structures, and macros. */
+#include "core_http_client.h"
+
+/* Include Secure Sockets implementation of transport interface. */
+#include "transport_secure_sockets.h"
```

Comment:
We need to move these headers to after `#include "logging_stack.h" otherwise the logging definitions in the config will supercede the ones in this test.